electrum/gui/qt/installwizard.py

```diff
@@ -690,7 +721,14 @@ def f(text):
         vbox.addWidget(line)
         vbox.addWidget(WWLabel(warning))
 
-        warn_issue4566_label = WWLabel(MSG_PASSPHRASE_WARN_ISSUE4566)
+        warn_issue4566_label = WWLabel(
+            _("Warning") + ": " +
+            _("You have multiple consecutive whitespaces or leading/trailing "
+                "whitespaces in your passphrase.") + " " +
+            _("This is discouraged.") + " " +
+            _("Due to a bug, old versions of ELCASH Wallet will NOT be creating the "
+                "same wallet as newer versions or other software.")
+        )
```

Comment:
don't we have anything like lazy gettext available in app? i liked all consts on top much more :(

Reply:
Unfortunately not